src/app/AppRoutes.tsx

```diff
@@ -122,4 +123,4 @@ const ConnectedRoutes = connect<StateProps, {}, {}, AppState>(
   })
 )(Routes);
 
-export default withRouter(ConnectedRoutes);
\ No newline at end of file
+export default withRouter(hot(ConnectedRoutes));
```

Comment:
Just a note, the `hot()` function is essentially a NOOP in production builds

See https://github.com/gaearon/react-hot-loader/blob/master/root.js